src/hooks/useGetWalletNfts.ts

```diff
@@ -0,0 +1,109 @@
+import { useWallet } from '@binance-chain/bsc-use-wallet'
+import { useEffect, useReducer } from 'react'
+import { getPancakeRabbitContract } from 'views/Nft/utils/contracts'
+
+const pancakeRabbitsContract = getPancakeRabbitContract()
+
+export type NftMap = {
+  [key: number]: {
+    tokenUri: string
+    tokenIds: number[]
+  }
+}
+
+type Action = { type: 'set_nfts'; data: NftMap } | { type: 'reset' }
+
+type State = {
+  isLoading: boolean
+  nfts: NftMap
+}
+
+const initialState: State = {
+  isLoading: true,
+  nfts: {},
+}
+
+const reducer = (state: State, action: Action) => {
+  switch (action.type) {
+    case 'set_nfts':
+      return {
+        ...initialState,
+        isLoading: false,
+        nfts: action.data,
+      }
+    case 'reset':
+      return {
+        ...initialState,
+        isLoading: false,
+      }
+    default:
+      return state
+  }
+}
+
+const useGetWalletNfts = () => {
+  const [state, dispatch] = useReducer(reducer, initialState)
+  const { account } = useWallet()
+
+  useEffect(() => {
+    const fetchNfts = async () => {
```

Comment:
This hook's logic is taken from the `NFTProvider`. Currently it provides a map of a wallet's NFTs

```
[bunnyId]: {
  tokenUri: string
  tokenIds: number[]
}
```